README.md

```diff
@@ -191,7 +191,7 @@ Silent mode off: `.o-ft-buttons--inverse`
 ### Deprecated
 
 The `--faux` modifier is now deprecated and will be removed in next major release.
-The `.o-ft-buttons__pagination` wasn't following the BEM convention properly and will be removed in next major release. Use `..o-ft-buttons-pagination` instead.
+The `.o-ft-buttons__pagination` wasn't following the BEM convention properly and will be removed in next major release. Use `.o-ft-buttons-pagination` instead.
```

Comment:
Well it's easy to have both `o-ft-` and `o-` variables cohabit, but classes and mixins are a lot of overhead.